src/main/java/frc/robot/sensors/IrSensor.java

```diff
@@ -0,0 +1,80 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+/**
```

Comment:
Nice touch but this is not needed since git has a version history of who modified what and when. Also it is misleading since someone else could come and change a few lines in this file.

Reply:
OK, will keep in mind for future commits. Won't change it for now since it would require the merge request to be re-approved, and the issue is fairly minor .